run.sh

```diff
@@ -265,6 +267,7 @@ if [ "${NETWORK}" == "regtest" ]; then
     else
         /srv/explorer/$DAEMON/bin/${DAEMON}d -conf=/data/.$DAEMON.conf -datadir=/data/$DAEMON -daemon
     fi
+    cli -rpcwait createwallet "" || true
```

Comment:
Something like `cli -rpcwait loadwallet "" || cli createwallet ""` would be better, so that it won't silently ignore errors other than 'this wallet already exists'.